### PR TITLE
docs: Added the malayalam documentation for pull request

### PR DESCRIPTION
**Pull request by vyshakputhusseri**
Only the malayalam version for documenting how to open a pull request
was opened.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: docs/i18n-languages/malayalam/how-to-work-on-coding-challenges.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+ഈ വെല്ലുവിളികൾ സൃഷ്ടിക്കുന്നതിന് വളരെയധികം സർഗ്ഗാത്മകതയും വിശദാംശങ്ങളിലേക്ക് ശ്രദ്ധയും ആവശ്യമാണ്. എന്നാൽ നിങ്ങൾക്ക് ധാരാളം സഹായം ലഭിക്കും. നിങ്ങൾക്ക് സംഭാവനകളുടെ ഒരു മുഴുവൻ ടീമിൽ നിന്നും പിന്തുണയുണ്ട്, നിങ്ങൾക്ക് ആശയങ്ങൾ ഉയർത്താനും നിങ്ങളുടെ വെല്ലുവിളികൾ ഡെമോ ചെയ്യാനും കഴിയും. [contributors room](https://gitter.im/freecodecamp/contributors) സജീവമായി തുടരുക കൂടാതെ ധാരാളം ചോദ്യങ്ങൾ ചോദിക്കുക.
+
+Wനിങ്ങളുടെ സഹായത്തോടെ, വരും വർഷങ്ങളിൽ ദശലക്ഷക്കണക്കിന് ആളുകളെ കോഡ് ചെയ്യാൻ പഠിക്കാൻ സഹായിക്കുന്ന ഒരു സംവേദനാത്മക കോഡിംഗ് പാഠ്യപദ്ധതി ഞങ്ങൾക്ക് രൂപകൽപ്പന ചെയ്യാൻ കഴിയും.
+
+ഓരോ വെല്ലുവിളിക്കുമുള്ള ഉള്ളടക്കം അതിന്റേതായ മാർക്ക്ഡൗൺ ഫയലിൽ സംഭരിച്ചിരിക്കുന്നു. ഈ മാര്കഡൗൺ  ഫയൽ പിന്നീട് ഞങ്ങളുടെ ഉപകരണങ്ങൾ ഉപയോഗിച്ച് നിങ്ങൾക്ക് സംവദിക്കാൻ കഴിയുന്ന വെബ്‌പേജിലേക്ക് മറയ്ക്കുന്നു. [`/curriculum`](/curriculum) directory ൽ നിങ്ങൾക്ക്‌  freecodecamp.org എല്ലാ പാഠ്യപദ്ധതിയും കണ്ടെത്താൻ‌ കഴിയും.
+
+ഈ വെല്ലുവിളികളിൽ നിങ്ങൾക്ക് പ്രവർത്തിക്കാൻ രണ്ട് വഴികളുണ്ട്: [`/curriculum`](/curriculum) directory.
+
+There are two ways you can work on these challenges:
```
Comment (thecodingaviator) [inline docs/i18n-languages/malayalam/how-to-work-on-coding-challenges.md]: This line has not been translated

File: docs/i18n-languages/malayalam/how-to-open-a-pull-request.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+`fix/feat/chore/refactor/docs/perf (scope): PR Title`
+
+ഒരു ഉദാഹരണം `fix(learn): Fixed tests for the do...while loop challenge`.
+
+| Scope | Documentation |
+|---|---|
+| `learn`,`curriculum` | For Pull Requests making changes to the curriculum challenges. |
+| `client` | For Pull Requests making changes to client platform logic or user interface |
+| `guide` | For Pull Requests which make changes to the guide. |
+| `docs` | For Pull Requests making changes to the project's documentation. |
```
Comment (thecodingaviator) [inline docs/i18n-languages/malayalam/how-to-open-a-pull-request.md]: Should these headings and descriptions be translated @raisedadead ?

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

File: docs/i18n-languages/malayalam/how-to-open-a-pull-request.md (hunk 2 above)
Comment (raisedadead) [reply docs/i18n-languages/malayalam/how-to-open-a-pull-request.md]: Yes.

File: docs/i18n-languages/malayalam/how-to-open-a-pull-request.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## എങ്ങനെ ഒരു പുൾ അഭ്യർത്ഥന (PR) നിർദ്ദേശിക്കാം
+
+1. എഡിറ്റുകൾ‌ ചെയ്‌തുകഴിഞ്ഞാൽ‌, നിങ്ങളുടെ ഫോർ‌ക്കിന്റെ GitHub പേജിൽ‌ ഒരു പുൾ‌ അഭ്യർ‌ത്ഥന സൃഷ്‌ടിക്കാൻ‌ നിങ്ങളോട് ആവശ്യപ്പെടും.
+
+    ![Image - Compare pull request prompt on GitHub](/docs/images/github/compare-pull-request-prompt.png)
+
+2. സ്ഥിരസ്ഥിതിയായി, എല്ലാ പുൾ അഭ്യർത്ഥനകളും ഫ്രീകോഡ്ക്യാമ്പ് പ്രധാന റിപ്പോ, `മാസ്റ്റർ` ബ്രാഞ്ചിന് എതിരായിരിക്കണം.
+    ഒരു പുൾ അഭ്യർത്ഥന ഉന്നയിക്കുമ്പോൾ നിങ്ങളുടെ അടിസ്ഥാന ഫോർക്ക് freeCodeCamp / freeCodeCamp ആയി സജ്ജമാക്കിയിട്ടുണ്ടെന്ന് ഉറപ്പാക്കുക.
+    ![Image - Comparing forks when making a pull request](/docs/images/github/comparing-forks-for-pull-request.png)
```
Comment (thecodingaviator) [inline docs/i18n-languages/malayalam/how-to-open-a-pull-request.md]: Please translate this line

File: docs/i18n-languages/malayalam/how-to-open-a-pull-request.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+| `learn`,`curriculum` | For Pull Requests making changes to the curriculum challenges. |
+| `client` | For Pull Requests making changes to client platform logic or user interface |
+| `guide` | For Pull Requests which make changes to the guide. |
+| `docs` | For Pull Requests making changes to the project's documentation. |
+
+## എങ്ങനെ ഒരു പുൾ അഭ്യർത്ഥന (PR) നിർദ്ദേശിക്കാം
+
+1. എഡിറ്റുകൾ‌ ചെയ്‌തുകഴിഞ്ഞാൽ‌, നിങ്ങളുടെ ഫോർ‌ക്കിന്റെ GitHub പേജിൽ‌ ഒരു പുൾ‌ അഭ്യർ‌ത്ഥന സൃഷ്‌ടിക്കാൻ‌ നിങ്ങളോട് ആവശ്യപ്പെടും.
+
+    ![Image - Compare pull request prompt on GitHub](/docs/images/github/compare-pull-request-prompt.png)
```
Comment (thecodingaviator) [inline docs/i18n-languages/malayalam/how-to-open-a-pull-request.md]: Please translate this line

File: docs/i18n-languages/malayalam/how-to-open-a-pull-request.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+# ഒരു പുൾ റിക്വസ്റ്റ്(PR) എങ്ങനെ തുറക്കാം
+
+## ഒരു നല്ല പുൾ റിക്വസ്റ്റ് ശീർഷകം എങ്ങനെ തയ്യാറാക്കാം:
+
+ഒരു പുൾ റിക്വസ്റ്റ (PR) തുറക്കുമ്പോൾ, ഇനിപ്പറയുന്ന ഫോർമാറ്റിൽ നിങ്ങളുടെ PR- ന് എന്ത് ശീർഷകം നൽകണമെന്ന് തീരുമാനിക്കാൻ ഇനിപ്പറയുന്ന സ്കോപ്പ് പട്ടിക ഉപയോഗിക്കുക:
+`fix/feat/chore/refactor/docs/perf (scope): PR Title`
+
+ഒരു ഉദാഹരണം `fix(learn): Fixed tests for the do...while loop challenge`.
+
+| Scope | Documentation |
```
Comment (thecodingaviator) [inline docs/i18n-languages/malayalam/how-to-open-a-pull-request.md]: Please translate this line

File: docs/i18n-languages/malayalam/how-to-open-a-pull-request.md (hunk 2 above)
Comment (thecodingaviator) [inline docs/i18n-languages/malayalam/how-to-open-a-pull-request.md]: Translate the documentation part on the right

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
check comments

**Pull request closed without merging**